Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml

```
@@ -109,7 +109,7 @@ script:
       name: alert_status
     - description: The aggregate alert id.
       name: aggregate_alert_id
-  dockerimage: demisto/sixgill:1.0.0.79377
+  dockerimage: demisto/sixgill:1.0.0.80180
```

Set the default incoming mapper for the integration:

```suggestion
  dockerimage: demisto/sixgill:1.0.0.80180
  defaultmapperin: classifier-mapper-incoming-Cybersixgill-Actionable-Alerts
```

Might be better to change the `id`, `name` properties of the classifier to something friendlier (e.g. _Cybersixgill Actionable Alerts Incoming Mapper_ and update the `defaultmapperin` here if you do.